feat: Add scroll listener to the nearest scrollable container and allow consumer to disable it (#3408)
excalidraw/excalidraw: src/packages/excalidraw/CHANGELOG.md

## Excalidraw API

> - Add scroll listener to the nearest scrollable container and allow consumer to disable it [#3408](https://github.com/excalidraw/excalidraw/pull/3408). By default scroll listener will be added to the nearest scrollable container so offsets get updated when position gets updated due to scroll. This behaviour can be disabled by setting `detectScroll` to false.

Reviewer: ```suggestion
- Recompute offsets on `scroll` of the nearest scrollable container [#3408](https://github.com/excalidraw/excalidraw/pull/3408). This can be disabled by setting [`detectScroll`]( https://github.com/excalidraw/excalidraw/blob/master/src/packages/excalidraw/README.md#detectScroll) to `false`.
```

Removed redundant statements + linked to readme
